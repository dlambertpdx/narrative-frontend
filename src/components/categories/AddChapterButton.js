import React from 'react';
import PropTypes from 'prop-types';
import EditFormModalChapter from './EditFormModalChapter';

const AddChapterButton = ({ show, hideModal, showModal }) => {
  return (
    <>
      <div>
        <EditFormModalChapter show={show} hideModal={hideModal}></EditFormModalChapter>
      </div>
      <button type='button' onClick={showModal}>Add New</button>
    </>
  );
};

AddChapterButton.propTypes = {
  show: PropTypes.bool,
  hideModal: PropTypes.func,
  showModal: PropTypes.func
};

export default AddChapterButton;
