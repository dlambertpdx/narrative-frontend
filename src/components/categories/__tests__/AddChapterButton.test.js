import React from 'react';
import { shallow } from 'enzyme';
import AddChapterButton from '../AddChapterButton';

describe('AddChapterButton component', () => {
  it('renders an add button', () => {
    const wrapper = shallow(<AddChapterButton handleAddClick={() => true} />);
    expect(wrapper).toMatchSnapshot();
  });
});