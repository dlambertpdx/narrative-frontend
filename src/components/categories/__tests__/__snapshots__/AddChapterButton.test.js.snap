// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddChapterButton component renders an add button 1`] = `
<Fragment>
  <div>
    <EditFormModalChapter />
  </div>
  <button
    type="button"
  >
    Add New
  </button>
</Fragment>
`;
